test/core/test_extract.py

```diff
@@ -107,11 +107,11 @@ def test_get_dataset_indexes_without_bounds(self):
         self._assert_get_dataset_indexes_works(dataset)
 
     def test_get_dataset_indexes_with_bounds_inverse_lat(self):
```

Comment:
```suggestion
   def test_get_dataset_indexes_with_bounds_inverse_y(self):
```

Reply:
`lat` is fine here, because `get_dataset_indexes` is still aware of `lat`, not `y`.